src/components/Loader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {StyleSheet} from 'react-native';
+import {View, ActivityIndicator} from 'react-native';
+
+//change the color of the loader according to your choice
+const colorString = '#E30062';
+
+export default function Loader() {
```

Comment:
Why we are making the default function? Are we going to write more function components in the same file?

Reply:
We aren't exporting more than a Loader here so hence default export